features/step_definitions/networking.rb

```diff
@@ -882,8 +882,22 @@
   ensure_admin_tagged
   node = node(node_name)
   host = node.host
-  step "the default interface on nodes is stored in the clipboard"
-  @result = host.exec_admin("ifconfig #{cb.interface}")
+  cb_ipaddr ||= "ip_address"
+  @result = admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.type}")
+  if @result[:success] then
+     networkType = @result[:response].strip
+  end
+  case networkType
+  when "OVNKubernetes"
+    step %Q/I run command on the node's ovnkube pod:/, table("| bash | -c | ip route show default |")
+  when "OpenShiftSDN"
+    step %Q/I run command on the node's sdn pod:/, table("| bash | -c | ip route show default |")
+  else
+    raise "unknown networkType"
+  end
+  def_inf = @result[:response].split("\n").first.split(/\W+/)[7]
+  logger.info "The node's default interface is #{def_inf}"
+  @result = host.exec_admin("ifconfig #{def_inf}")
   cb[cb_ipaddr]=@result[:response].match(/\d{1,3}\.\d{1,3}.\d{1,3}.\d{1,3}/)[0]
```

Comment:
@anuragthehatter I've seen this before...perhaps we should think about putting it into a method call #ip_addr or something.  @akostadinov, suggestion as to where we might to put this?  base_helper.rb?

Reply:
If internal IP of node is different from IP reported by `oc get node -o yaml`, then we can add method `Node#internal_ip`. Good suggestion.

I'm just curious why node ip reported by API should be different from what we get in this way.

Reply:
@akostadinov @pruan-rht Internal IP of node is "not" different from IP reported by oc get node -o yaml. I don't think so we support node.ip etc hence this was was designed for use specifically for networking use cases.

Reply:
@anuragthehatter, ok, I see...  Let's merge this for now and we can put a TODO for enhancement later.